controller/comment-controller.js

```diff
@@ -0,0 +1,31 @@
+const Comment = require("../models/comment");
+const Posts = require("../models/posts");
+const validation = require("./validation");
+
+exports.addComment = async (req, res) => {
+  const { postId } = req.params;
+  const isValid = validation.isValidObjectId(postId);
+  if (!isValid) {
+    res.json({
+      err: "Invalid Id",
+    });
+    return;
+  }
+  const { description } = req.body;
+  const userId = req.user._id;
```

Comment:
You can make these `req` assignments at the top of the function body.

Reply:
updated